farm/modeling/adaptive_model.py

```diff
@@ -813,3 +718,223 @@ def load_from_adaptive_model(cls, adaptive_model):
 
     def forward(self, *batch):
         return super().forward(input_ids=batch[0], padding_mask=batch[1], segment_ids=batch[2])
+
+
+class Converter:
```

Comment:
How about we move this class to farm.conversion.transformers? adaptive_model.py is already huge